src/Plugin/formatter/FormatterJsonApi.php

```diff
@@ -251,4 +252,32 @@ protected function getRequest() {
     return restful()->getRequest();
   }
 
+  /**
+   * Move the embedded resources to the included key.
```

Comment:
This file is the JSON API formatter. Anything in this file only applies to that.